flake8_use_fstring/format.py

```diff
@@ -21,10 +21,7 @@ def __getitem__(self, i: int) -> bool:
             return False
         if next_token.exact_type != _token.NAME:
             return False  # pragma: no cover (syntax error)
-        if next_token.string != 'format':
-            return False
-
-        return True
+        return next_token.string == 'format'
```

Comment:
This is not desirable as the function uses the "return False conditions until True" style.